fetcher/src/job/job.controller.ts

```diff
@@ -20,13 +20,17 @@ export class JobController {
 
   async onModuleInit() {
     const chain = process.env.CHAIN
-    const aggregators = await loadActiveAggregators({ chain, logger: this.logger })
-
-    for (const aggregator of aggregators) {
+    const activeAggregators = await this.activeFetchers()
```

Comment:
How about changing the names somehow? The function is called `activeFetchers` but the assigned variable is `activeAggregators`? It would be nicer to unify them, either using word aggregators or fetchers to mitigate confusion.